Cargo.toml

```diff
@@ -17,6 +17,8 @@ ark-ec = { version = "0.2" }
 ark-ff = { version = "0.2" }
 ark-serialize = { version = "0.2" }
 ark-ed-on-bls12-377 = {version = "0.2", features = ["r1cs"]}
+digest = { version = "0.9.0" }
+lazy_static = { version = "1.4.0" }
```

Comment:
https://docs.rs/once_cell/1.8.0/once_cell/ might be preferable to `lazy_static`, since it avoids hiding the inner type behind the macro.

Reply:
(There's no real problem with using `lazy_static` though, if it turns out that the `once_cell` API is worse for some reason).